src/main/java/com/direwolf20/buildinggadgets/common/tools/CraftingConditionDestruction.java

```diff
@@ -11,7 +11,7 @@
     @Override
     public BooleanSupplier parse(JsonContext jsonContext, JsonObject jsonObject) {
         final boolean result;
```

Comment:
```suggestion
         final boolean result = InGameConfig.enableDestructionGadget;
```
??

Reply:
I didn't actually read, what I was changing there (I just switched to the InGameConfig). 
I agree that this code is senseless, wonder how it got here

Reply:
The other condition does the same, but why?!?